build.gradle

```diff
@@ -33,7 +33,7 @@ nexusStaging {
 }
 
 
-def hadoopVersion = "2.7.3"
+def hadoopVersion = "2.10.0"
```

Comment:
Upgrade the minor version of tony as well. This doesn't seem backward compatible. 

0.3.35 -> 0.4.0

Reply:
+1